### ISSUE TITLE
Segmentation layer add button is missing

### ISSUE THREAD

**Issue by fedorov** [Kitware/VolView#512]
<img width="1319" alt="image" src="https://github.com/Kitware/VolView/assets/313942/28585741-dd03-493d-88d9-fef1c716625b">

<img width="221" alt="image" src="https://github.com/Kitware/VolView/assets/313942/76a2e267-b1b1-4719-847f-3f91cc3da5dd">

**Comment by fedorov** [comment]
I see this option is available via the "kabob" menu. Some color overlay shows up, but not in the right place.

<img width="1320" alt="image" src="https://github.com/Kitware/VolView/assets/313942/ec26c7e0-53a7-43ce-80c7-fc52d4059dd5">

For comparison, same study in OHIF v2

https://viewer.imaging.datacommons.cancer.gov/viewer/1.2.840.113654.2.55.100481428377517059587592697630638604217

<img width="1328" alt="image" src="https://github.com/Kitware/VolView/assets/313942/38363dca-4107-4696-b9bd-ea5bc0946c26">

and OHIF v3

https://viewer.imaging.datacommons.cancer.gov/viewer/1.2.840.113654.2.55.100481428377517059587592697630638604217

<img width="1318" alt="image" src="https://github.com/Kitware/VolView/assets/313942/eb850772-ef00-4392-9eb0-38d82fff75fc">

**Comment by floryst** [comment]
Yes, I had moved the layering button into the menu for future functionality.

The segmentation alignment issue is being investigated and will hopefully be addressed soon.

**Comment by sedghi** [comment]
We encountered an alignment issue, and this code snippet resolved it, i hope it helps

https://github.com/cornerstonejs/cornerstone3D/blob/main/packages/adapters/src/adapters/Cornerstone/Segmentation_4X.js#L1583
